src/backend/vulkan/src/command.rs

```diff
@@ -28,7 +28,7 @@ fn map_subpass_contents(contents: com::SubpassContents) -> vk::SubpassContents {
 }
 
 fn map_buffer_image_regions<T>(
-    image: &n::Image,
+    _image: &n::Image,
```

Comment:
You should remove unused argument then.

Reply:
Nah, it's fine to have it still, since we'll want to do some checks on the image (e.g. is the range contained, does it support mapping, etc)